directory.json

```diff
@@ -13,6 +13,7 @@
 	"BP": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/India/BangPypers-api.json",
 	"CALLUG":"https://raw.githubusercontent.com/fossasia/fossasia-communities/master/India/LinuxUsersGroup-Calcutta-api.json",
 	"CDLug": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/China/ChengduGNU-LinuxUserGroupCDLug-api.json",
+	"Ceylon": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Sri%20Lanka/Ceylon-api.json",
```

Comment:
@jig08 I google the org before merging and it seems pretty open source to me. Check [this](http://www.ceylonlinux.com) out !!